### PR TITLE
chore: update to angular2 .52

### PR DESCRIPTION
**Pull request by Foxandxss**
This PR will make me cry all day and probably the rest of the week also. Worst update ever.

### CODE REVIEW

File: src/dropdown/dropdown.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import {Directive, Input, HostListener} from 'angular2/core';
 
-@Directive({selector: '[ngb-dropdown]', exportAs: 'ngbDropdown', host: {'[class.open]': 'open'}})
+@Directive({selector: '[ngbDropdown]', exportAs: 'ngbDropdown', host: {'[class.open]': 'open'}})
```
Comment (pkozlowski-opensource) [inline src/dropdown/dropdown.ts]: do we need / want to change those?

Comment (Foxandxss) [inline src/dropdown/dropdown.ts]: Do I want? No, I don't want any of this, this is all BS. Do we need to do it? Yes.

Comment (pkozlowski-opensource) [inline src/dropdown/dropdown.ts]: Can't we leave a selector as `ngb-dropdown`? I'm not saying we should, just didn't test so don't know if technically possible...

Comment (Foxandxss) [inline src/dropdown/dropdown.ts]: We can't.

Comment (wesleycho) [inline src/dropdown/dropdown.ts]: That was a breaking change as far as I understand - it makes me sad, as it is more natural to understand than this camelCase :( .

Comment (Foxandxss) [inline src/dropdown/dropdown.ts]: Completely agree. But we have to move on and merge this.

Comment (pkozlowski-opensource) [inline src/dropdown/dropdown.ts]: I thought that we can still use `ngb-dropdown` in a selector....

Comment (Foxandxss) [inline src/dropdown/dropdown.ts]: dash-stuff for element selectors, attributes ones are camelCase.

Comment (pkozlowski-opensource) [inline src/dropdown/dropdown.ts]: I would like to check this, since it sounds wrong to me...

Comment (Foxandxss) [inline src/dropdown/dropdown.ts]: please do, you're the one with connections :P

**Pull request closed without merging**